src/guides/v2.4/graphql/queries/pickup-locations.md

```diff
@@ -125,7 +143,7 @@ Use the `AreaInput` to apply filtration by distance. All attributes are required
 Attribute | Data type | Description
 --- | --- | ---
 `radius` | Int! | The radius to the search, in kilometers.
-`search_term` | String! | A combination of either the region, city, or postcode, a colon, and the country code. This value determines the location to use as the center of the search radius.  For example, `Austin:US`.
+`search_term` | String! | A combination of either the region, city, or postcode, a colon, and the country code. This value determines the location to use as the center of the search radius.  Valid search term - `Texas:US`|`Austin:US`|`78740:US`. After the colon, the two-letter Country code is required to be passed in uppercase.
```

Comment:
I'm not sure what you're trying to say here. Those | symbols can't be in a markdown table. 

Perhaps reword to 

Valid search terms include `Texas:US`, `Austin:US`, and `78740:US`. The two-letter country code must be uppercase